tests/end-to-end/test_database.py

```diff
@@ -73,6 +73,9 @@ def write_configuration(
           hostname: mongodb
           username: root
           password: test
+    sqlite_databases:
+        - name: sqlite_test
+          path: /tmp/sqlite_test.db
```

Comment:
Unless I'm missing it, it doesn't look like `/tmp/sqlite_test.db` is actually populated anywhere. So when the test dumps the database file, it's simply dumping.. nothing..? (I just confirmed that `sqlite3` will happily dump a non-existent file.) So do you think it's worth trying to create this as a SQLite test file within the test, so it's actually dumping something?

Related question: Do you think borgmatic should warn or error if the `path` file to be dumped doesn't exist? Or should that be on the user to worry about?

Reply:
For the first part, I don't think the contents of the file really matter for the test?
Second part, yeah, I value UX and think that a simple log should be done to let the user know that the file didn't exist firsthand, and they are creating an empty database and dumping it's contents.

Reply:
Okay, I'm fine with dumping nothing if you are! And a warning sounds great.